src/controllers/keyboard/layoututils.h

```diff
@@ -0,0 +1,63 @@
+#ifndef MIXXX_LAYOUTUTILS_H
+#define MIXXX_LAYOUTUTILS_H
+
+#include <QString>
+#include "control/controlobject.h"
+
+struct KbdKeyChar {
+    char16_t character;
+    bool is_dead;
+};
+
+// Representation of one <keyseq> element, parented to <control> elements
+struct KbdControllerPresetKeyseq {
+    // String representation of the key sequence (for example: Ctrl+a),
+    // found in the text of the <keyseq> element
+    QString keysequence;
+
+    // Keyboard layout at which this key sequence is targeted (for example: es_ES),
+    // found in the 'lang' attribute of the <keyseq> element
+    QString lang;
+
+    // Scancode following the scancode-set 2:
+    // https://www.win.tue.nl/~aeb/linux/kbd/scancodes-1.html#ss1.4
+    // The scancode is extracted from the 'scancode' attribute of the <keyseq> element
+    QString scancode;
+
+    // Indicates whether this key sequence should be translated or not. For example,
+    // key sequences like 'Ctrl+F1', 'Left', 'Shift+Right' or 'Space' do not need
+    // translation. In those cases, final will be true. It is extracted from the
+    // 'final' attribute of the <keyseq> element, where '1' => true and everything
+    // else => false (if no final attribute => false).
+    //
+    // Note: Final key sequences usually don't specify neither lang nor scancode.
+    bool final;
```

Comment:
It is hard to understand what "final" does. It looks like it is used for keys that are key label depended and not position dependent right?  It could be "moveWithLable" or something. Or we could invert the logic and make this one the default. 

`<keyseq>Shift+F10</keyseq>`

It is somehow obvious, that with no other attributes set, this is bound to the label and will be move.